eip/EIPS/eip-4337.md

```diff
@@ -343,19 +343,21 @@ Exceptions to the forbidden opcodes:
 2. `GAS` is allowed if followed immediately by one of { `CALL`, `DELEGATECALL`, `CALLCODE`, `STATICCALL` }.
    (that is, making calls is allowed, using `gasleft()` or `gas` opcode directly is forbidden)
 
-### Reputation scoring and throttling/banning for paymasters
+### Reputation scoring and throttling/banning for paymasters, deployers and aggregators
 #### Rationale
 
-Paymasters might influence many UserOperations in the mempool.
-To prevent abuse, we throttle down (or completely ban for a period of time) a paymaster that causes invalidation of large number of UserOperations in the mempool.
-To prevent paymasters from "sybil-attack", we require a paymaster to stake with the system, and thus make such DoS attack very expensive.
+UserOperation's memory access rules prevent them from interfere with each other.
+But "global" entities - paymasters, deployers and aggregators are accessed by multiple UserOperations, and thus might influence them.
```

Comment:
Maybe make it a bit more specific - that it could actually invalidate these previously-valid ops.
"...and thus might invalidate multiple previously-valid UserOperations."